server/conn.go

```diff
@@ -729,6 +729,7 @@ func insertDataWithCommit(goCtx goctx.Context, prevData, curData []byte, loadDat
 		if !reachLimit {
 			break
 		}
+		terror.Log(loadDataInfo.Ctx.StmtCommit())
```

Comment:
If StmtCommit meets error, should we return the error?

Reply:
Other places only log the StmtCommit error too.
We can discuss how to treat this error later.

Reply:
Just `terror.Log` it now, I'll handle it later. @shenli

Reply:
ok